source/local/local.go

```diff
@@ -268,6 +273,19 @@ func getFeaturesFromFiles() (map[string]string, error) {
 
 		// Append features
 		fileFeatures := parseFeatures(lines)
+
+		// Check expiration of file features
+		expiryDate, err := getExpirationDate(lines)
```

Comment:
Hmm, I just realized that we don't have support for `# comment` lines in the feature files (I thought we had). I think that should be added (inside `parseFeatures()`, I guess) , otherwise it tries to interpret the `# +expiry-time: ...` line as a feature label, too, and throw some errors in the logs.

You could make parsing `# +<some-tag>: <value>` (I don't know whether they should be called tags or directives or something else) a generic function that updates an opts struct or smth. We might want to add other tags/directives later.

Also, you could do this parsing inside `parseFeatures()` so that the tag/directive could effect only after it's encountered. This would add some flexibility. E.g. in the case of expiry-time there could be some features that don't expire and others that do (even have different expiry times). I don't know if this is needed, and we can have expiry-time as a file-wide global setting, but for some others we might choose differently.

WDYT?